### ISSUE TITLE
yml file is wrong?

### ISSUE THREAD

**Issue by FMagnani** [mims-harvard/SubGNN#23]
Hi everyone, 
do someone has the same issue? 
I create the environment as
`conda env create --file SubGNN.yml `
but when I test the code using one of the Real-World dataset they provide, as 
`python train_config.py -config_path config_files/ppi_bp/ppi_bp_config.json`
a ton of errors starts appearing. I'm troubleshooting them one by one, but I don't want to report them all now. Instead, I believe that it's easier to get an updated and verified yml file from someone that could made it work.

So guys, if you do have a conda environment in which the code runs, please share your yml file, super thanks!

**Comment by FMagnani** [comment]
I could get something, I leave it here for anyone interested in order to make him save some time.
My tip is to run the following:
`conda create --name <name of the conda environment>`
`conda activate <name of the conda environment>`
`conda install pytorch torchvision torchaudio cudatoolkit=11.1 -c pytorch-lts -c nvidia`
`conda install pyg -c pyg -c conda-forge`
`conda install matplotlib`
`pip install pytorch-lightning`
`conda install -c bioconda fastdtw`
In this way the core file of the repo works - I mean SubGNN.py in which the model is defined.

Then, to employ also the file train_config.py, run the following:
`conda install -c conda-forge optuna`
`conda install -c conda-forge commentjson`

I hope this helps.

**Comment by jacbam** [comment]
Hi did you figure it out? I'm facing the same issue

**Comment by FMagnani** [comment]
Nope, I could not solve it... Dependencies can be really annoying
